ui/js/selectors/wallet.js

```diff
@@ -60,14 +66,14 @@ export const selectTransactionItems = createSelector(
           //value on transaction, amount on outpoint
           //amount is always positive, but should match sign of value
           const amount = parseFloat(
-            item.amount ? (item.value < 0 ? -1 : 1) * item.amount : item.value
+            item.amount ? item.balance_delta : item.value
```

Comment:
This is a very confusing line. Does it have to work this way? I check if I have an `amount`, and if I do, I use `balance_delta`, but if I don't, I use `value`?

If this is correct, perhaps something should be changed in the APIs to make this less confusing/complex.